<?xml version="1.0"?>
<launch>

  <node pkg="task_nodes" type="turn_and_stop" name="turn_and_stop" output="screen" >
    <remap from="cmd_vel" to="/vehicle/cmd_vel" />
    <remap from="stop_line_dist" to="/lane_camera/stop_line_dist" />
    <param name="sensitive_range" value="3.0" />
    <param name="angle_change_thres" value="1.30" />
    <param name="turn_direction" value="left" />
  </node>
  
  <include file="$(find task_nodes)/launch/startup/live_lane_navigation.launch" />
  
  <node pkg="rqt_reconfigure" type="rqt_reconfigure" name="rqt_reconfigure" />
  
  <node pkg="rviz" type="rviz" name="sequoia_nav_laser_rviz" respawn="true" args="-d $(find sequoia_nav_laser)/rviz/rviz_config.rviz" />

</launch>
